### DISCUSSION TITLE
sidebar/sanfene/weifuwu

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#259] [Announcements]
# sidebar/sanfene/weifuwu

下载次数超 1 万次，1.3 万字 33 张手绘图，详解 33 道微服务（Dubbo、Spring Cloud）面试高频题（让天下没有难背的八股），面渣背会这些八股文，这次吊打面试官，我觉得稳了（手动 dog）

https://javabetter.cn/sidebar/sanfene/weifuwu.html

**Comment by Lweiis** [comment]
"14.能说下HTTP和RPC的区别吗？
HTTP 和 RPC 区别"
Markdown 表格出现了格式错乱？

**Comment by Kddk-K** [comment]
9.12一刷

**Comment by hachinekooo** [comment]
1.16

**Comment by hachinekooo** [comment]
mark
